components/switch/README.md

```diff
@@ -110,3 +102,7 @@ If an activity is set to `Visible` but also has other conditions affecting its v
   </ul>
 </d2l-switch-visibility>
 ```
+
+## Accessbility
+
+- When `text-position` is set to `hidden`, the required `text` will be used for the switch's hidden label so that it is [accessible](https://www.w3.org/WAI/WCAG22/Techniques/aria/ARIA14) to non-sighted users.
```

Comment:
Did you mean to use (from your comment) https://www.w3.org/WAI/WCAG22/quickref/#name-role-value instead? I think referencing the WCAG criteria itself is more valuable since it highlights the importance of the value, rather than the specific technique we've chosen, which could change any time.

Reply:
Yep, I copied the link from the wrong tab 😅 I'll replace it with that one 😄